kivy/atlas.py

```diff
@@ -388,30 +397,38 @@ def create(outname, filenames, size, padding=2, use_path=False):
         return outfn, meta
 
 
-if __name__ == '__main__':
-    """ Main line program. Process command line arguments
-    to make a new atlas. """
+def run(argv):
```

Comment:
Can you let the run() under the __main__ test ?
One of the experimentation i'm doing is to strip python files of non-reachable code when shipping to production. Everything in __main__ except the main entrypoint is never executed on mobile, and strip down.

Thanks :)